flecs_ecs_sys/src/flecs.h

```diff
@@ -10176,25 +10176,25 @@ int ecs_log_last_error(void);
 
 
 /* Handle addon dependencies that need declarations to be visible in header */
-#ifdef FLECS_MONITOR
-#ifndef FLECS_STATS
-#define FLECS_STATS
-#endif
-#ifndef FLECS_SYSTEM
-#define FLECS_SYSTEM
-#endif
-#ifndef FLECS_TIMER
-#define FLECS_TIMER
-#endif
-#endif
-
-#ifdef FLECS_REST
-#define FLECS_HTTP
-#endif
-
-#ifdef FLECS_PLECS
-#define FLECS_EXPR
-#endif
+//#ifdef FLECS_MONITOR
+//#ifndef FLECS_STATS
+//#define FLECS_STATS
+//#endif
+//#ifndef FLECS_SYSTEM
+//#define FLECS_SYSTEM
+//#endif
+//#ifndef FLECS_TIMER
+//#define FLECS_TIMER
+//#endif
+//#endif
+//
+//#ifdef FLECS_REST
+//#define FLECS_HTTP
+//#endif
+//
+//#ifdef FLECS_PLECS
+//#define FLECS_EXPR
+//#endif
```

Comment:
Is this because you get re-definition warnings? (This should be changed upstream to be more like:

```
#if defined(FLECS_PLECS) && !defined(FLECS_EXPR)
#define FLECS_EXPR
#endif
```

Reply:
yup indeed, maybe I'll PR this soon-ish to flecs